newrelic-fluent-bit-output.yaml

```diff
@@ -12,7 +12,7 @@ environment:
       - build-base
       - busybox
       - ca-certificates-bundle
-      - go
+      - go-1.20
```

Comment:
Comment should be added here to explain the pin; link to upstream bug or this pr; and to explain what needs to change to unpin this.